apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -1,17 +1,28 @@
 import { useEffect, useState } from 'react';
 import { Box, Typography, Link } from '@mui/material';
-import { Public as PublicIcon } from '@mui/icons-material';
+import {
+  ArrowBackIosNew as ArrowBackIosNewIcon,
+  Public as PublicIcon,
+} from '@mui/icons-material';
 import {
   Twc2SupportDetail,
   FacebookLinksSection,
+  SupportDetailStrings,
+  getSupportDetailStrings,
   getTwc2Detail,
 } from '../services';
 import { Container } from '../common/components';
 import { PrimaryButton } from '../common/components/RoundedButton';
 import './SupportDetailWip.css';
 
-function Header() {
-  return <nav className="support-header">Navigation</nav>;
+function Header(props: { title: string }) {
+  const { title } = props;
+  return (
+    <nav className="support-header">
+      <ArrowBackIosNewIcon />
+      <Typography variant="h4">{title}</Typography>
```

Comment:
Is this meant to have the same font type as the blurb title? I don't have any frontend sense but was just wondering if the header font should be different from the blurb.

Reply:
That's fine for now, we can fine tune it later. We can cross check against the mocks

Reply:
Ditto, we'll tweak these in the next round of changes :)